doc/content/guides/semtech-interop/_index.md

```diff
@@ -0,0 +1,93 @@
+    join-euis:
+      - 000e000000000000/16
+      - 000f000000000000/16
+
+  - file: ./ttn/js.yaml
+    join-euis:
+      - 70b3d57ed0000000/40
+```
+
+A configuration like this would make Join-Requests for JoinEUIs starting with `000E` or `000F` to be handled by Join Server defined at `semtech/js.yaml` in interoperability mode and all other ones to be handled by Join Server defined at `tti/js.yaml`.
+The prefix specified for the Semtech Join Server should match the JoinEUI of the devices you would like to use Semtech Join Server.
+NOTE: All paths are relative to the file they're defined in, that is `semtech/js.yaml` defined in `interopconf/config.yaml` is expected to be located at `interopconf/semtech/js.yaml`.
+NOTE(2): Interoperability Join Server requests are only performed if either:
+a. Cluster-local Join Server is not available(e.g. it's disabled in the stack or it's down)
+b. Cluster-local Join Server is available, but does not contain the information about a device indentified by particular JoinEUI:DevEUI pair.
+NOTE(3): `0000000000000000/0` prefix matches all JoinEUIs, while being the least specific, hence it always matches last. Use this prefix if you wish to define a "fallback" Join Server interoperability configuration.
+
+## Join Server Configuration
+
+The Join Server configuration provides means to configure how the components interact with the Join Server.
+The configuration supports multiple options, however the options required for Semtech Join Server in particular, are:
+- `fqdn` - the FQDN to use (e.g. `jstest.sm.tc`)
+- `port` - the port to use (e.g. `7009`)
+- `headers` - HTTP headers to send, defined as key-value map.(e.g. `Authorization: AgK:9W2zoXa8xQqsb4alP2AgSJoaT2NT`)
+- `tls` - TLS configuration to use
+  - `root-ca` - path to client CA
+  - `certificate` - path to client TLS certificate
+  - `key` - path to client TLS key
+- `protocol` - the protocol to use - one of `BI1.0` or `BI1.1`(e.g. `BI1.0`)
+- `paths` - the URI paths to use for various requests. Default paths are empty.
+  - `join` - the URI path to use for `JoinReq` (e.g. `api/v1/rens/rens-1::3/lbi_joinreq`)
+
+
+For example:
+```bash
+$ mkdir interopconf/semtech
+$ touch interopconf/semtech/js.yaml
+```
+
+`interopconf/semtech/js.yaml` content:
+```yaml
+fqdn: jstest.sm.tc
+port: 7009
+protocol: BI1.0
+paths:
+  join: api/v1/rens/rens-1::3/lbi_joinreq
+tls:
+  root-ca: clientca.pem
+  certificate: clientcert.pem
+  key: clientkey.pem
+headers:
+  Authorization: AgK:9W2zoXa8xQqsb4alP2AgSJoaT2NT
+```
+
+NOTE: All paths in the example are relative to `interopconf/semtech`.
+
+## Stack configuration
+
+Each component of the stack has its own interoperability configuration, however sharing configuration in-between components is allowed.
+In order to let Network Server and Application Server use the interoperability configuration we just defined above:
+```
+  ttn-lw-stack start --ns.interop.directory interopconf --as.interop.directory interopconf
```

Comment:
This probably can be configured via the stack config as well - how do I do that?
What's the name of option corresponding to `ns`? @johanstokking

Reply:
What do you mean? Env is `TTN_LW_NS_INTEROP_DIRECTORY` right?

I would highly recommend not explicitly starting the stack this way, but rather follow what happens in the Getting Started, i.e. using Docker and environment keys in the `docker-compose.yml`

Reply:
I had an impression there's a `yaml` config available for the stack as well, which is what I meant.
I will change this to `docker-compose` + env variables